cc/algorithms/partition-selection-strategies_test.cc

```diff
@@ -15,28 +15,359 @@
 //
 
 #include "algorithms/partition-selection-strategies.h"
+#include "algorithms/numerical-mechanisms-testing.h"
 
-#include "gmock/gmock.h"  //TODO do I need this?
-#include "gtest/gtest.h"  //TODO do I need this?
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
 
 namespace differential_privacy {
 namespace {
 
-//TODO literally just nabbed this from numerical-mechanisms_test.cc, fix later
-/*using testing::_;
-using testing::DoubleEq;
-using testing::DoubleNear;
+using testing::_;
 using testing::Eq;
-using testing::Ge;
+using testing::DoubleNear;
+using testing::DoubleEq;
 using testing::MatchesRegex;
-using testing::Return;
 
-class PartitionSelectionStrategiesTest : public ::testing::Test {};*/
+//PreaggregationPartitionSelection Tests
+
+TEST(PartitionSelectionStrategiesTest, PreaggPartitionSelectionUnsetEpsilon) {
+  	PreaggPartitionSelection::Builder test_builder;
+  	auto failed_build = test_builder.SetDelta(0.1)
+  									.SetMaxPartitionsContributed(2).Build();
```

Comment:
Nit: Is the overabundance of whitespace here and below an artifact of GitHub, or is it actually in the code? If the latter, perhaps it should be shortened to be more sensible?

Reply:
I think GitHub may have doubled the size of the indent or something?  SetDelta and SetMaxPartitionsContributed look vertically aligned in my local copy >__<

Reply:
I wouldn't worry too much about it. Fry the bigger fish first, and if you don't have time to figure this out, I say don't worry about it.